### PR TITLE
Updated version and CHANGELOG.md

### PR DESCRIPTION
**Pull request by fboucquez**

### CODE REVIEW

**Review by segfaultxavi — APPROVED** [review]
No problems with the text.

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,24 @@ All notable changes to this project will be documented in this file.
 
 The changelog format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
+## [v2.2.0] - 3-Dec-2020
+### Added
+- Catapult-server finality (0.10.0.4) support.
```
Comment (rg911) [inline CHANGELOG.md:8]: Catapult-server finality (0.10.0.4 - testnet/v3) would be clearer

File: version.txt [hunk 2]
```diff
@@ -1 +1 @@
-2.1.1
+2.2.0
```
Comment (rg911) [inline version.txt:1]: should we use version suffix to differentiate between main & testnet?
e.g. 2.2.0-tesetnet

**Review by rg911 — COMMENTED** [review]
we should make a

File: version.txt (hunk 2 above)
Comment (fboucquez) [reply version.txt:1]: I wouldn't add the testnet prefix/suffix. Server and libs are not adding it.

**Pull request merged**